framework/src/media/utils/MediaUtils.cpp

```diff
@@ -538,15 +535,16 @@ bool createWavHeader(FILE *fp)
 	ret = fwrite(header, sizeof(unsigned char), WAVE_HEADER_LENGTH, fp);
 	if (ret < 0) {
```

Comment:
i think `ret != WAVE_HEADER_LENGTH ` it's better.

Reply:
I modify and upload another PR #2588

Reply:
good